toolchains/nodejs/WORKSPACE

```diff
@@ -31,8 +31,14 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
     name = "build_bazel_rules_nodejs",
-    sha256 = "f10a3a12894fc3c9bf578ee5a5691769f6805c4be84359681a785a0c12e8d2b6",
-    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/5.5.3/rules_nodejs-5.5.3.tar.gz"],
+    sha256 = "dcc55f810142b6cf46a44d0180a5a7fb923c04a5061e2e8d8eb05ccccc60864b",
+    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/5.8.0/rules_nodejs-5.8.0.tar.gz"],
+)
+
+http_archive(
+    name = "rules_nodejs",
+    sha256 = "08337d4fffc78f7fe648a93be12ea2fc4e8eb9795a4e6aa48595b66b34555626",
+    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/5.8.0/rules_nodejs-core-5.8.0.tar.gz"],
 )
```

Comment:
[CI fails with not up-to-date docs](https://github.com/tweag/rules_nixpkgs/actions/runs/4024688026/jobs/6917002150#step:5:1086)
And running
```
bazel run //docs:update-README.md
```
fails with 
```
The following chain of repository dependencies lead to the missing definition.
 - @rules_nodejs
This could either mean you have to add the '@rules_nodejs' repository with a statement like `http_archive` in your WORKSPACE file (note that transitive dependencies are not added automatically), or move an existing definition earlier in your WORKSPACE file.
```
Indeed, `@rules_nodejs` is only listed in `toolchains/nodejs/WORKSPACE` but not imported in the top-level workspace. @avdv what's the best place to import that at this point, taking the factoring out of the docs workspace into account?

@kczulko is this actually needed? As I understand the `rules_nodejs` [setup instructions](https://github.com/bazelbuild/rules_nodejs/releases/tag/5.8.0) this is an optional thing if one wants only the core, or is it mandatory?

Reply:
> @kczulko is this actually needed? As I understand the rules_nodejs [setup instructions](https://github.com/bazelbuild/rules_nodejs/releases/tag/5.8.0) this is an optional thing if one wants only the core, or is it mandatory?

Is this (rules_nodejs (core)) really needed? Maybe not. However, now I am using its PLATFORMS "enumeration" to do the mapping between nixpkgs <-> rules_nodejs platforms. This as well can be ~copy-pasted to rules_nixpkgs repo - just wanted to avoid boilerplate duplication.

Reply:
I see, I thought the core package would be a subset, but the platforms dict is only included in core. Okay, in that case both will have to be added as dependencies.

Reply:
With the factoring out of the docs workspace, one should just run `bazel run //docs:update-README.md` inside the appropriate toolchain directory.